SQF/dayz_code/stringtable.xml

```diff
@@ -5393,7 +5393,7 @@
 		</Key>
 		<Key ID="STR_ITEMSODA_FULL_CLEAN_CODE_DESC_6">
 			<English>A clean, unopened can of Mikhail Made Lemonade. Made by Mikhail.</English>
-			<German>Eine saubere, ungeöffnete Dose "Mikhail Made"-Limonade. Hergestellt von Mikhail.</German>
+			<German>Eine saubere, ungeöffnete Dose "Mikhail Made"-Limonade.</German>
```

Comment:
Please add back "Hergestellt von Mikhail". Im not sure it is a inscription for a modder or a team member of the dayz mod.